### PR TITLE
React Google Maps

### PR DESCRIPTION
**Pull request by k3ntako**

### CODE REVIEW

File: client/formatters/EventFormatter.js [hunk 1]
```diff
@@ -12,11 +12,17 @@ const timeFormatOptions = {
...
     const datetime = new Date(event.datetime);
     const formattedDate = datetime.toLocaleDateString([], dateFormatOptions);
     const formattedTime = datetime.toLocaleTimeString([], timeFormatOptions);
     return `${formattedDate}, ${formattedTime}`
+  },
+  formatDate: function(datetime){
+    return datetime.toLocaleDateString([], dateFormatOptions);
+  },
+  formatTime: function(datetime){
+    return datetime.toLocaleDateString([], dateFormatOptions);
```
Comment (mienaikoe) [inline client/formatters/EventFormatter.js]: haha I did this in my PR already

File: client/pages/AddEvent/EventMap.css [hunk 2]
```diff
@@ -1,33 +1,34 @@
...
-  margin: 5px 0 !important;
+  margin: 5px 0;
   text-overflow: ellipsis;
   outline: none;
   font-size: 15px;
   background-color: #fff;
 }
 
-.addEventTextarea{
+textarea.addEventInput{
```
Comment (mienaikoe) [inline client/pages/AddEvent/EventMap.css]: why is it important to provide both qualifiers?

File: client/pages/AddEvent/EventMapContainer.js [hunk 3]
```diff
@@ -1,4 +1,6 @@
 import React, {Component} from 'react';
+import { Input } from 'react-toolbox/lib/input';
```
Comment (mienaikoe) [inline client/pages/AddEvent/EventMapContainer.js]: to get react-toolbox-themr styles, you need to write this as:
`import Input from 'react-toolbox/lib/input/Input'`

Yea it sucks

File: client/pages/AddEvent/EventMap.css (hunk 2 above)
Comment (k3ntako) [reply client/pages/AddEvent/EventMap.css]: I changed the textarea to a React Toolbox Input. I will delete this.

File: client/pages/AddEvent/EventMapContainer.js (hunk 3 above)
Comment (k3ntako) [reply client/pages/AddEvent/EventMapContainer.js]: Should I change the buttons, datepicker and timepicker as well?

Comment (mienaikoe) [reply client/pages/AddEvent/EventMapContainer.js]: Oh - Yes, if they take in color.

**Pull request merged**